src/styles/main.css

```diff
@@ -0,0 +1,10 @@
+/* The integration's default injected base.css file */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  body {
+    @apply bg-off-white font-satoshi;
+  }
+}
```

Comment:
Would it work to add `bg-off-white font-satoshi` to the index.astro file in that `body`?